src/mad_spark_alt/core/robust_json_handler.py

```
@@ -0,0 +1,197 @@
+            rf'"{key}"\s*:\s*(true|false|null)',
+            rf'{key}\s*:\s*"([^"]*)"',
+        ]
+        
+        for pattern in patterns:
+            match = re.search(pattern, text, re.IGNORECASE)
+            if match:
+                value = match.group(1)
+                try:
+                    # Try to parse as JSON
+                    result[key] = json.loads(value)
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)